src/pubsub.js

```diff
@@ -0,0 +1,13 @@
+let subscribers = []
+
+export const subscribe = fn => {
+  subscribers = [...subscribers, fn]
```

Comment:
For my own curiosity, is there a benefit over doing `.concat()`?

Reply:
Nah, not really.